src/elements/content-sidebar/additional-tabs/AdditionalTabs.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @flow
+ * @file Preview sidebar additional tabs component
+ * @author Box
+ */
+
+import React, { PureComponent } from 'react';
+import AdditionalTab from './AdditionalTab';
+import AdditionalTabsLoading from './AdditionalTabsLoading';
+
+import './AdditionalTabs.scss';
+
+type Props = {
+    tabs: ?Array<AdditionalSidebarTab>,
+};
+
+type State = {
+    isLoading: boolean,
+};
+
+class AdditionalTabs extends PureComponent<Props, State> {
+    numLoadedTabs: number = 0;
+
+    initialState: State = {
+        isLoading: true,
+    };
+
+    constructor(props: Props) {
+        super(props);
+
+        this.state = { ...this.initialState };
```

Comment:
It doesn't look like we ever reset state to the initial value, so you could move state instantiation out of the constructor.